shared/js/project.js

```diff
@@ -274,7 +274,14 @@ var projects = module.exports = {
     });
   },
   serverSideRemix: function() {
-    location.href = projects.getPathName('remix');
+    if (current && !current.name &&
```

Comment:
Never mind, we set "Untitled Project" in `init()` if the project doesn't already have a name.